.gitignore

```diff
@@ -1 +1,5 @@
 .idea/
+model-runner
+model-runner.sock
+model-runner
+model-runner.sock
```

Comment:
Duplicate?

Reply:
I think he's referring to the existing surrounding lines in the `.gitignore`.  It looks like `model-runner` and `model-runner.sock` are already in that file.